src/sidebar/components/Annotation/AnnotationEditor.tsx

```diff
@@ -169,6 +172,14 @@ function AnnotationEditor({
 
   const mentionsEnabled = store.isFeatureEnabled('at_mentions');
   const usersWhoAnnotated = store.usersWhoAnnotated();
+  const focusedGroupMembers = store.getFocusedGroupMembers();
+
+  useEffect(() => {
+    // When entering edit mode, load focused group members if not yet loaded
+    if (mentionsEnabled && focusedGroupMembers === null) {
+      groupsService.loadFocusedGroupMembers();
+    }
+  }, [focusedGroupMembers, groupsService, mentionsEnabled]);
```

Comment:
I'd prefer not to add this as a side effect, but this component is loaded both when creating a new annotation or editing an existing one, so it's convenient to centralize the logic here.